json-core/src/main/java/io/micronaut/json/JsonMapper.java

```diff
@@ -90,15 +92,15 @@ default <T> T readValueFromTree(@NonNull JsonNode tree, @NonNull Class<T> type)
     <T> T readValue(@NonNull InputStream inputStream, @NonNull Argument<T> type) throws IOException;
 
     /**
-     * Parse and map json from the given stream.
-     *
-     * @param inputStream The input data.
-     * @param type        The type to deserialize to.
-     * @param <T>         Type variable of the return type.
-     * @return The deserialized object.
-     * @throws IOException IOException
+     * Read a value from the given input stream for the given type.
+     * @param inputStream The input stream
+     * @param type The type
+     * @param <T> The generic type
+     * @return The value or {@code null} if it decodes to null
+     * @throws IOException If an unrecoverable error occurs
```

Comment:
add `@since` to all the methods

Reply:
i added `@since` to the new default methods https://github.com/micronaut-projects/micronaut-core/pull/9424/commits/fe96d570c6b63f167f55ebca130e340207e6f983